pandas/errors/__init__.py

```diff
@@ -12,6 +12,9 @@
     OutOfBoundsTimedelta,
 )
 
+from pandas._version import NotThisMethod
```

Comment:
Just realized this is from our vendored version of versioneer which the user should never see. In the pre-commit check could you ignore `versioneer.py`?

Reply:
I believe that's coming from `pandas/_version.py`, but yeah I can ignore `pandas/_version.py`